src/common/globals.js

```diff
@@ -265,13 +267,20 @@ define([
             valueType: 'section'
         });
 
-        const storageMetadata = Storage.getAvailableBackends().map(id => Storage.getStorageMetadata(id));
+        let backends = Storage.getAvailableBackends();
+
+        if(excludeFromStructure === 'dataHash') {
```

Comment:
Splitting this into two plugins would also make this code more clear. I understand that what it is really saying is that creating an artifact from existing data is not possible with the GME backend because the GME backend stores by hash and cannot look up files by path but this is not clear from this check.